### PR TITLE
Rework Installation into Distribution Channels

### PR DESCRIPTION
**Pull request by RomanTsukanov**
[Preview on my machine](http://tsukanov-w10:4446/Documentation/20_1/Guide/Common/Distribution_Channels/npm/)

### CODE REVIEW

**Review by AlexJameson — APPROVED** [review]

File: concepts/Common/Distribution Channels/05 CDN/10 Subresource Integrity Hashes.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-You can use <a href="https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity" target="_blank">Subresource Integrity</a> (SRI) when fetching DevExtreme files from CDN. Below are the files' SRI hashes for DevExtreme minor releases. Each file has a `sha384` and `sha512` hash.
+You can use <a href="https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity" target="_blank">Subresource Integrity</a> (SRI) hashes to verify DevExtreme files when fetching them from CDN. Each link below contains `sha384` and `sha512` hashes:
 
 ---
 ##### 20.1.0
```
Comment (BingoRUS) [inline concepts/Common/Distribution Channels/05 CDN/10 Subresource Integrity Hashes.md:4]: Why we have this part of the article? We haven't published any files of v20.1 to CDN yet... @viterobk

File: concepts/Common/Distribution Channels/05 CDN/05 CDN.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+## Scripts
```
Comment (BingoRUS) [inline concepts/Common/Distribution Channels/05 CDN/05 CDN.md:3]: This article looks kinda weird. Just a list of all files available via CDN will follow (see [the example](https://js.devexpress.com/Documentation/Guide/Getting_Started/Installation/CDN_Services/#Subresource_Integrity_Hashes) of Subresource Integrity Hashes). We had this part of the article to explain which files are needed in different cases, now we don't have this information. I think we should keep at least the first paragraph from the previous article.

File: concepts/Common/Distribution Channels/01 npm/01 npm.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+npm is most often used for delivering DevExtreme to Angular, Vue, and React applications. The following packages are provided:
```
Comment (dirk-pieterse) [inline concepts/Common/Distribution Channels/01 npm/01 npm.md]:
```suggestion
npm is used to deliver DevExtreme to Angular, Vue, and React applications. The following packages are provided:
```

File: concepts/Common/Distribution Channels/01 npm/10 Yarn.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+All DevExtreme npm packages can be installed using Yarn:
```
Comment (dirk-pieterse) [inline concepts/Common/Distribution Channels/01 npm/10 Yarn.md]:
```suggestion
You can use Yarn to install all DevExtreme npm packages:
```

File: concepts/Common/Distribution Channels/05 CDN/05 CDN.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
+DevExtreme delivers the following files via CDN:
```
Comment (dirk-pieterse) [inline concepts/Common/Distribution Channels/05 CDN/05 CDN.md]: Why 'delivers'?

File: concepts/Common/Distribution Channels/10 NuGet.md [hunk 6]
```diff
@@ -0,0 +1,10 @@
+When using Visual Studio, you can get the DevExtreme scripts and stylesheets from the <a href="https://www.nuget.org/packages/DevExtreme.Web/" target="_blank">DevExtreme Web</a> NuGet package. Run the following command in the <a href="https://docs.microsoft.com/en-us/nuget/consume-packages/install-use-packages-powershell" target="_blank">Package Manager Console</a> to install this package:
```
Comment (dirk-pieterse) [inline concepts/Common/Distribution Channels/10 NuGet.md]:
```suggestion
When you use Visual Studio, you can get the DevExtreme scripts and stylesheets from the <a href="https://www.nuget.org/packages/DevExtreme.Web/" target="_blank">DevExtreme Web</a> NuGet package. Run the following command in the <a href="https://docs.microsoft.com/en-us/nuget/consume-packages/install-use-packages-powershell" target="_blank">Package Manager Console</a> to install this package:
```

File: concepts/Common/Distribution Channels/15 ZIP Archive.md [hunk 7]
```diff
@@ -0,0 +1,3 @@
+The ZIP archive is a suitable choice if you [use jQuery](/concepts/58%20jQuery%20Components/05%20Add%20DevExtreme%20to%20a%20jQuery%20Application/10%20Local%20Files.md '/Documentation/Guide/jQuery_Components/Add_DevExtreme_to_a_jQuery_Application/#Local_Files'), develop applications with DevExtreme on Mac or Linux, or simply want to try DevExtreme. The archive contains DevExtreme scripts, stylesheets, and demos.
```
Comment (dirk-pieterse) [inline concepts/Common/Distribution Channels/15 ZIP Archive.md]:
```suggestion
The ZIP archive can be used if you [use jQuery](/concepts/58%20jQuery%20Components/05%20Add%20DevExtreme%20to%20a%20jQuery%20Application/10%20Local%20Files.md '/Documentation/Guide/jQuery_Components/Add_DevExtreme_to_a_jQuery_Application/#Local_Files'), develop applications with DevExtreme on Mac or Linux, or simply want to try DevExtreme. The archive contains DevExtreme scripts, stylesheets, and demos.
```

File: concepts/Common/Distribution Channels/20 Installer for Windows.md [hunk 8]
```diff
@@ -0,0 +1,5 @@
+The installer for Windows provides Visual Studio integration features, such as project scaffolding. Use it if you develop applications in Visual Studio.
+
+After the installation, you will be able to find DevExtreme sources in the %ProgramFiles(x86)%\DevExpress 20.1\DevExtreme folder.
```
Comment (dirk-pieterse) [inline concepts/Common/Distribution Channels/20 Installer for Windows.md]:
```suggestion
After the installation, you can find the DevExtreme sources in the %ProgramFiles(x86)%\DevExpress 20.1\DevExtreme folder.
```

**Review by dirk-pieterse — APPROVED** [review]

**Pull request merged**